kitsune/users/models.py

```diff
@@ -86,6 +86,7 @@ class Profile(ModelBase, SearchMixin):
     is_fxa_migrated = models.BooleanField(default=False)
     fxa_uid = models.CharField(blank=True, null=True, unique=True, max_length=128)
     fxa_avatar = models.URLField(max_length=512, blank=True, default='')
+    has_subscription = models.BooleanField(default=False)
```

Comment:
Maybe convert this to `has_subscriptions` to keep this similar to the claim name.